lib/beard_shaver.rb

```diff
@@ -0,0 +1,9 @@
+class BeardShaver
```

Comment:
"lib" is a bad place for this. This class is part of business logic which is specific for this app, so it should live in "app" folder. Lib is for logic which is not specific for the app and is assumed to not change frequently (that's why it is not auto reloaded in Rails by default).